src/ui/components/shared/MaterialIcon.tsx

```diff
@@ -29,16 +26,12 @@ export default function MaterialIcon({
   iconSize = "base",
   ...rest
 }: MaterialIconProps) {
-  const fontLoading = useSelector((state: UIState) => selectors.getFontLoading(state));
   return (
     <div
       {...rest}
       className={classnames(
         "leading-none",
         className,
-        {
-          invisible: fontLoading,
-        },
```

Comment:
I think this commit landed separately, but once this 2nd omnibus lands, it'd be nice to get back into a cadence for smaller PRs :)

Reply:
Yep, agreed!